alvr/client_core/src/graphics/lobby.rs

```diff
@@ -217,8 +227,18 @@ impl LobbyRenderer {
         });
 
         let render_targets = [
-            super::create_gl_swapchain(device, &swapchain_textures[0], view_resolution),
-            super::create_gl_swapchain(device, &swapchain_textures[1], view_resolution),
+            super::create_gl_swapchain(
+                device,
+                &swapchain_textures[0],
+                view_resolution,
+                TextureFormat::Rgba8Unorm,
+            ),
+            super::create_gl_swapchain(
+                device,
+                &swapchain_textures[1],
+                view_resolution,
+                TextureFormat::Rgba8Unorm,
```

Comment:
Why explicitly put rgba8 here instead of just using `DEFAULT_FORMAT`?

Reply:
Also in favor of being specific here, since we want to be explicit about the fact that SDR uses RGB8 and HDR uses RGB16

Reply:
@shinyquagsire23 should i rename the constant to SDR_FORMAT?

Reply:
wouldn't hurt to have it defined that way tbh, though I guess technically the format is whatever the OpenXR runtime says it supports (I think rgba8 or brga8 was a required one)